src/sentry/static/sentry/app/views/onboarding/configure/waiting.jsx

```diff
@@ -1,34 +1,52 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {t} from 'app/locale';
+import styled from 'react-emotion';
+
+import Button from 'app/components/button';
+import HookStore from 'app/stores/hookStore';
 
 class Waiting extends React.Component {
   static propTypes = {
     skip: PropTypes.func,
     hasEvent: PropTypes.bool.isRequired,
+    params: PropTypes.object,
   };
 
   render() {
     return (
       <div className="awaiting-event">
-        <div className="pull-right">
-          <div className="btn btn-primary" onClick={this.props.skip}>
-            {t('All done!')}
-          </div>
-        </div>
-        <div className="wrap waiting-text">
-          {!this.props.hasEvent ? (
-            <h3 className="animated-ellipsis">{t('Waiting to receive an error')}</h3>
-          ) : (
-            <h3>{t("You've successfully sent an error")}</h3>
-          )}
-          <div className="robot">
-            <span className="eye" />
+        <div className="row">
+          <div className="col-sm-10">
+            <div className="wrap waiting-text">
+              {!this.props.hasEvent ? (
+                <h3 className="animated-ellipsis">{t('Waiting to receive an error')}</h3>
+              ) : (
+                <h3>{t("You've successfully sent an error")}</h3>
+              )}
+              <div className="robot">
+                <span className="eye" />
+              </div>
+            </div>
           </div>
+          <CenteredButtons className="col-sm-2">
+            <div className="pull-right">
+              <Button priority="primary" data-test-id="configure-done" onClick={this.props.skip}>
+                {t('All done!')}
+              </Button>
+              {HookStore.get('experiment:sample-event').length
```

Comment:
You've both brought up really good points @adhiraj & @billyvg. To summarize: a downside to this current approach is we still have the concept of `experiment` exposed in sentry. Forking the entire component with `hookOrDefault` is not ideal but has the benefit of completely containing all aspects of experiments in getsentry. The major downside is a change can happen to a component in Sentry that would break in getsentry side and tests wouldn't catch it afaict. This seems bad. If so, can we mitigate the badness somehow? I'm good with merging with this current approach as we keep discussing. Thanks both for your feedback.

Reply:
For now- going to use a generic hookname :)